src/components/player-info.js

```diff
@@ -37,7 +37,8 @@ AFRAME.registerComponent("player-info", {
   schema: {
     avatarSrc: { type: "string" },
     avatarType: { type: "string", default: AVATAR_TYPES.SKINNABLE },
-    muted: { default: false }
+    muted: { default: false },
+    sharingAvatarCamera: { default: false }
```

Comment:
nit: Variable name could signal that it's a boolean (`isSharingAvatarCamera` or `hasAvatarCamera`, etc). I know we don't always do this (the line above has `muted` instead of `isMuted`)